src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -57,6 +57,10 @@ public function __construct(iterable $voters = [], string $strategy = self::STRA
      */
     public function decide(TokenInterface $token, array $attributes, $object = null)
     {
+        if (\count($attributes) > 1) {
+            @trigger_error('Passing more than one Security attribute to '.__METHOD__.' is deprecated since Symfony 4.4. Use multiple decide() calls or the expression language (e.g. "has_role(...) or has_role(...)") instead.', \E_USER_DEPRECATED);
```

Comment:
why not also use __METHOD__ for the `decide()` in the second part of the error message
same for `AuthorizationChecker.php`

Reply:
I did that in order to not create a too long deprecation message. I'm often lost in the FQCNs in the deprecation messages in our projects. Using the FQCN once was a comprise of adding information without lengthening the message.